src/parser/SpatialQuery.cpp

```diff
@@ -29,27 +30,62 @@ void SpatialQuery::addParameter(const SparqlTriple& triple) {
   } else if (predString == "numNearestNeighbors") {
     if (!object.isInt()) {
       throw SpatialSearchException(
-          "The parameter <numNearestNeighbors> expects an integer (the maximum "
+          "The parameter `<numNearestNeighbors>` expects an integer (the "
+          "maximum "
           "number of nearest neighbors)");
     }
     maxResults_ = object.getInt();
   } else if (predString == "maxDistance") {
     if (!object.isInt()) {
       throw SpatialSearchException(
-          "The parameter <maxDistance> expects an integer (the maximum "
+          "The parameter `<maxDistance>` expects an integer (the maximum "
           "distance in meters)");
     }
     maxDist_ = object.getInt();
   } else if (predString == "bindDistance") {
     setVariable("bindDistance", object, distanceVariable_);
+  } else if (predString == "joinType") {
+    if (!object.isIri()) {
+      // This 'if' is redundant with `extractParameterName`, but we want to
+      // throw a more precise error description
```

Comment:
This comment is strangely worded and I don't understand it. Please say more clearly what you mean. If this is about code (or message) duplication, maybe just go ahead and avoid it